kola/tests/podman/podman.go

```diff
@@ -169,9 +169,9 @@ func podmanWorkflow(c cluster.TestCluster) {
 
 		found := false
 		for _, container := range psInfo.containers {
-			if container.ID == id {
+			if container.ID == id[0:12] {
```

Comment:
It looks like every time `id` is used we're only using this range, should we just initially set it to that value rather than selecting a limited set? (e.x.: `id = string(out)[0:12]`)

Reply:
:+1: I didn't realize I changed them all